dirty_cat/test/test_fuzzy_join.py

```diff
@@ -1,8 +1,11 @@
 import pandas as pd
 import pytest
 
-from dirty_cat import fuzzy_join, print_worst_matches
-from dirty_cat.experimental import enable_fuzzy_join  # noqa
+# isort : off
+from dirty_cat.experimental import enable_fuzzy_join  # noqa, isort: skip
+from dirty_cat import fuzzy_join, print_worst_matches  # isort: skip
+
+# isort : on
```

Comment:
Do you need `isort: skip` if you use `isort: on/off`?

Reply:
It seems the answer is yes. I tried all possible cases and it needed to stay to make it work.